INSTALL.rst

```diff
@@ -33,7 +33,7 @@ gcc
 """
 - 4.9 - 7 (if you want to build of Nvidia GPUs, supported compilers depend on your current `CUDA version <https://gist.github.com/ax3l/9489132>`_)
 
-  - CUDA 8.0: Use gcc 4.9
+  - CUDA 8.0: Use gcc 4.9 - 5.3
   - CUDA 9.0 - 9.1: Use gcc 4.9 - 5
```

Comment:
@psychocoderHPC I think this one should also be 4.9 - 5.3 (CUDA 9.2 fixes GCC 5.4/6 std::tuple issues afaik)

Reply:
std::tuples should not work with CUDA 9.0 and 9.1

> From the CMAKE side all gcc 5.X are allowed with CUDA 9.

I guess it's the CUDA "reference side" you are talking about